src/persist/src/file.rs

```diff
@@ -115,9 +116,11 @@ impl FileBuffer {
 
             if len > 0 {
                 let mut buf = [0u8; 8];
-                buffer_file.seek(SeekFrom::End(8))?;
+                buffer_file.seek(SeekFrom::End(-8))?;
```

Comment:
🤦🏾 whoops sorry

Reply:
Don't be, it passed the buffer tests! Now there are just more tests